sdk/devcenter/developer-devcenter-rest/README.md

```diff
@@ -50,6 +50,9 @@ After setup, you can choose which type of [credential](https://github.com/Azure/
 As an example, [DefaultAzureCredential](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/identity/identity#defaultazurecredential)
 can be used to authenticate the client.
 
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
```

Comment:
With the Spring Grove effort going on, we probably shouldn't add info of using secerts to authenticate?

Reply:
Thanks! Any suggestions to improve here?

Reply:
Makes sense to me. It's not a must have here. I agree to remove it.

Reply:
removed